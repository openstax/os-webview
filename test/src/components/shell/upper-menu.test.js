import UpperMenu from '~/components/shell/header/upper-menu/upper-menu';

describe('UpperMenu', () => {
    const p = new UpperMenu();

    it('creates', () => {
        expect(p.el.innerHTML).toBeTruthy();
    });
    it('matches snapshot', () => {
        console.log('To update snapshot: node_modules/.bin/jest --updateSnapshot --testNamePattern=UpperMenu');
        expect(p.el.innerHTML).toMatchSnapshot();
    });
});
