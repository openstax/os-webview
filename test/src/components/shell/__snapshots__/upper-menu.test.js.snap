// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpperMenu matches snapshot 1`] = `"<a class=\\"nav-menu\\" href=\\"/bookstore-suppliers\\">Bookstores</a><a class=\\"nav-menu\\" href=\\"/impact\\">Our Impact</a><a class=\\"nav-menu\\" href=\\"/foundation\\">Supporters</a><a class=\\"nav-menu\\" href=\\"/give\\">Give</a><a class=\\"nav-menu\\" href=\\"https://openstax.secure.force.com/help\\">Help</a><a class=\\"logo rice-logo logo-wrapper\\" href=\\"http://www.rice.edu\\"><img src=\\"/images/rice.png\\" alt=\\"Rice University logo\\" height=\\"30\\"></a>"`;
